cmd/decrypt.go

```diff
@@ -17,24 +19,38 @@ var decryptCmd = &cobra.Command{
 	RunE:    DecryptCommand,
 }
 
+var allowFail bool
+
 func init() {
 	RootCmd.AddCommand(decryptCmd)
+	decryptCmd.Flags().BoolVarP(&allowFail, "allow-fail", "a", false, "Do not fail if not all files can be decrypted")
```

Comment:
This shorthand feels a bit weird - `a` feels like 😄 .

Maybe consider removing the shorthand entirely?